### ISSUE TITLE
Rails 5 deprecation warnings

### ISSUE THREAD

**Issue by jamesmk** [wearefine/fae#167]
Here are some fun deprecation warnings I get when running the tests on Rails 5. Not urgent we but should eventually address.

```
DEPRECATION WARNING: Passing the separator argument as a positional parameter is deprecated and will soon be removed. Use `separator: '_'` instead. (called from block in ___ibrary__eb_erver__ocuments_work_fine__fae_app_views_fae_shared__form_header_html_slim__3258569578465688496_70346386054760 at /Library/WebServer/Documents/work/fine/_fae/app/views/fae/shared/_form_header.html.slim:34)

DEPRECATION WARNING: ActiveModel::Errors#get is deprecated and will be removed in Rails 5.1. To achieve the same use model.errors[:name]. (called from get at /Users/james/.rvm/gems/ruby-2.3.1@fae/gems/activemodel-5.0.0.1/lib/active_model/errors.rb:124)

DEPRECATION WARNING: use_transactional_fixtures= is deprecated and will be removed from Rails 5.1 (use use_transactional_tests= instead) (called from <top (required)> at /Library/WebServer/Documents/work/fine/_fae/spec/requests/wines_spec.rb:25)

DEPRECATION WARNING: Passing string to define callback is deprecated and will be removed in Rails 5.1 without replacement. (called from <class:ReleaseNote> at /Library/WebServer/Documents/work/fine/_fae/spec/dummy/app/models/release_note.rb:19)

DEPRECATION WARNING:           [Devise] including `Devise::TestHelpers` is deprecated and will be removed from Devise.
          For controller tests, please include `Devise::Test::ControllerHelpers` instead.
 (called from <top (required)> at /Library/WebServer/Documents/work/fine/_fae/spec/controllers/fae/application_controller_spec.rb:3)
```

thanks

**Comment by tshedor** [comment]
@jamesmk Looks like you added this commit into master? Is this issue all good then?

**Comment by Spone** [comment]
See #273 for a fix to Rails 5.1 deprecation warnings.

**Comment by jamesmk** [comment]
Addressing the remaining warnings in v2.0.
